python/nammu/controller/NammuController.py

```diff
@@ -538,16 +538,17 @@ def process_server_response(self, oracc_log, request_log, autolem):
         """
         # Check if there were any validation errors and pass them to the
         # ATF area to refresh syntax highlighting.
-        self.process_validation_errors(oracc_log)
+
         if oracc_log:
```

Comment:
Good catch! This is fixed in commit 640f2f7, and solves issue #238, which I have now closed.